templates/content/basic/src/MyProvider.Runtime/MyProvider.Runtime.fsproj

```diff
@@ -1,42 +1,22 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<Project>
-  <!-- See https://stackoverflow.com/questions/43921992/how-can-i-use-beforebuild-and-afterbuild-targets-with-visual-studio-2017 -->
-  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
-  <Import Project="..\..\netfx.props" />
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <OutputType>Library</OutputType>
-    <TargetFrameworks>netstandard2.0; net45</TargetFrameworks>
-    <DisableImplicitFSharpCoreReference>true</DisableImplicitFSharpCoreReference>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <FSharpToolsDirectory>typeproviders</FSharpToolsDirectory>
```

Comment:
What is this "FSharpToolsDirectory" thing being set?

Reply:
This creates the exact same layout that was specified before. It is a property that exists in the F# compiler targets (and apparently has been there a very long time). @KevinRansom showed it to me and it greatly simplifies this requirement

Reply:
> This creates the exact same layout that was specified before. It is a property that exists in the F# compiler targets (and apparently has been there a very long time). @KevinRansom showed it to me and it greatly simplifies this requirement

Yes, that is good, and it packages the dependencies too.  At least there's some advantage to that dubious cross-project reference :-)